client/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "trabe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "client.js",
+  "type": "module",
+  "scripts": {
+    "start": "nodemon exec node client.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "latest",
+    "cheerio": "latest",
+    "express": "^4.18.2",
+    "fs": "latest",
+    "node-fetch": "^3.3.2",
+    "nodemon": "^3.0.3",
```

Comment:
Strictly speaking, nodemon is a `devDependency`. Is not really needed to run the client.